### PR TITLE
RIFEX-54/Lumino-Channels-Detail

### PR DESCRIPTION
**Pull request by rorolopetegui**
This PR adds the lumino channels detail page

Lumino-Channels-Details-resized

### CODE REVIEW

File: ui/app/rif/components/lumino/open-channel/index.js [hunk 1]
```diff
@@ -6,28 +6,55 @@ import rifActions from '../../../actions';
...
   static propTypes = {
-    tokenAddress: PropTypes.string.isRequired,
-    tokenNetworkAddress: PropTypes.string.isRequired,
-    tokenName: PropTypes.string.isRequired,
-    tokenSymbol: PropTypes.string.isRequired,
+    tokenAddress: PropTypes.string,
+    tokenNetworkAddress: PropTypes.string,
+    tokenName: PropTypes.string,
+    tokenSymbol: PropTypes.string,
+    //
```
Comment (marcosmartinez7) [inline ui/app/rif/components/lumino/open-channel/index.js]: Remove this

File: ui/app/rif/components/luminoNetworkChannels.js [hunk 2]
```diff
@@ -117,6 +122,15 @@ function mapStateToProps (state) {
...
+    ...params,
+    tabOptions: {
+      hideTitle: true,
+      showSearchbar: false,
+      showBack: true,
+      letSelectedIndex: params.letSelectedIndex,
+    },
+  })),
     getChannelsGroupedByNetwork: () => dispatch(rifActions.getChannelsGroupedByNetwork()),
     getChannel: () => {},
```
Comment (marcosmartinez7) [inline ui/app/rif/components/luminoNetworkChannels.js]: What is the purpose of this function without body?

File: ui/app/rif/components/tabs/index.js [hunk 3]
```diff
@@ -10,6 +10,8 @@ class Tabs extends Component {
     onChange: PropTypes.func,
     showBack: PropTypes.bool,
     backAction: PropTypes.func,
+    classes: PropTypes.any,
+    letSelectedIndex: PropTypes.number,
```
Comment (marcosmartinez7) [inline ui/app/rif/components/tabs/index.js]: Why is this prop needed?

File: ui/app/rif/pages/index.js [hunk 4]
```diff
@@ -106,7 +106,9 @@ function buildTabScreen (screenName, context, dispatch) {
       <Tabs tabs={tabs}
             onChange={(tab) => onTabChange(tab)}
             showBack={tabOptions.showBack}
-            backAction={() => dispatch(rifActions.navigateBack())}/>
+            backAction={() => dispatch(rifActions.navigateBack())}
+            letSelectedIndex={context.params.tabOptions.letSelectedIndex}
```
Comment (JONAF2103) [inline ui/app/rif/pages/index.js]: You have tabOptions.tabIndex for this

**Review by JONAF2103 — CHANGES_REQUESTED** [review]

File: ui/app/rif/components/tabs/index.js (hunk 3 above)
Comment (rorolopetegui) [reply ui/app/rif/components/tabs/index.js]: I deleted, cause i didnt notice that we can set this in taboptions

**Review by JONAF2103 — APPROVED** [review]
LGTM

**Pull request merged**